src/main.py

```
@@ -1,17 +1,63 @@
 from fastapi import FastAPI
 import uvicorn
 import os
+import database
```

Ruff (0.8.2)

`database` imported but unused

Remove unused import: `database`

(F401)